quickstart/src/main/java/com/microsoft/azure/kusto/quickstart/SampleApp.java

```diff
@@ -268,35 +283,63 @@ public class SampleApp {
     private static boolean waitForUser;
 
     public static void main(String[] args) {
+        createKustoTracer();
```

Comment:
Make it like python - make it off by default, put it in a way that is easy to comment out, etc

Reply:
Done